src/main/scala/diplomaticobjectmodel/model/OMAddressing.scala

```diff
@@ -63,6 +65,22 @@ case class OMRegFieldGroup (
   _types: Seq[String] = Seq("OMRegFieldGroup", "OMCompoundType")
 ) extends OMCompoundType
 
+case class OMAddressBlock (
+  name: String,
+  addressOffset:  BigInt,
```

Comment:
This should be `baseAddress` to match the [duh](https://github.com/sifive/duh-schema/blob/4cf25992b38bdece9ecde92fe3e864713bbe3eef/lib/memoryMap.js#L24) and [IP-XACT](http://www.accellera.org/XMLSchema/IPXACT/1685-2014/memoryMap.xsd) terminology.